### DISCUSSION TITLE
Welcome to epicseven Discussions!

### DISCUSSION THREAD

**Discussion by Apen** [Apen/epicseven#31] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ds108j** [comment]
Hi ! (Bonjour).

I just discovered your tool, and it seems that you lack the last characters.
If you need my help, I can help you updating the characters JS files (if it is manual, or maybe you extract it from another source ?)

If you need help, just ask !

**Reply by Apen** [reply]
Hi :-) It is from an API wich is not updated for the moment (https://api.epicsevendb.com/). Just need a few days i think.
Thanks

**Reply by ds108j** [reply]
You're welcome !

**Reply by Apen** [reply]
Its done 😁
